Applications/Telemetry/telemetry.ini

```diff
@@ -8,5 +8,5 @@ schemaname=db.sql
 
 [telemetry]
 units=1
-unit0call=REPLACEME
-unit0id=REPLACEME
\ No newline at end of file
+unit0call= REPLACEME
```

Comment:
I am open to suggestions @hdkmike , I like the idea of sticking towards what originally seems to be the case of no spaces. Example [wikipedia](https://en.wikipedia.org/wiki/INI_file)